### PR TITLE
[DATA-755] Update webcam config

### PR DESCRIPTION
**Pull request by kharijarrett**
Motivated by (https://docs.google.com/document/d/18p3LV6ghH-m0xGjO7MZRz4zK0X8JePmGZRAXBtm0k7E/edit)

### CODE REVIEW

**Review by bhaney — APPROVED** [review]
Looks good! But hold off on merging this until the end of the week, so we can coordinate with other teams.

File: components/camera/videosource/webcam.go
Comment (bhaney) [inline components/camera/videosource/webcam.go:141]: I think with "Shown but not required" this means that this should NOT be omitempty. 

@Fahmina or @nataliajacobowitz do you know how it will end up looking?

**Review by Fahmina — CHANGES_REQUESTED** [review]
Lets chat w devin before we merge these changes

**Review by Fahmina — APPROVED** [review]
LGTM!

**Pull request merged**
